app/models/discussion.rb

```diff
@@ -8,12 +8,16 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
+  after_create :find_or_create_subscriptions
   after_touch :notify_new_comment
 
   private
 
+  def find_or_create_subscriptions
+    Subscriber.for_discussion(self).subscribe(User.all)
+  end
+
   def notify_new_comment
-    subscriptions = Subscriber.for_discussion(self).subscribe(User.all)
-    subscriptions.map(&:new_comment)
+    find_or_create_subscriptions.map(&:new_comment)
   end
 end
```

Comment:
1. This was already handled previously. The subscription will get created, yes. But it's default state is `unwatched`, which means no notifications, no emails, etc. It only changes states when you comment the discussion yourself

2. True. There's also no way to subscribe to a discussion without commenting on it. That, I believe, was discussed at the time we first talked about this, and was intentionally left out for now.
The reasoning was that we could eventually add that (maybe a simple watch/unwatch button on the discussion) if we felt the need